BEIMA.Backend/StorageService/AzureStorageProvider.cs

```diff
@@ -54,7 +54,7 @@ public async Task<string> PutFile(IFormFile file)
                     await client.UploadAsync(stream);
                 }                   
                 return fileUid;
-            } catch (Exception e)
+            } catch (Exception)
```

Comment:
Should we add some basic console logging for debugging purposes?

Reply:
Will be handled by a future task that is focused on adding logging/writing to a file on exceptions.